### PR TITLE
[python] Add simulation to python example.

### PR DESCRIPTION
**Pull request by chrisdembia**
This PR adds simulation to the python example. This PR also adds display geometry. I also fixed a small number of bugs/typos.

When the example is run now, it'll pop up the API Visualizer.

Unfortunately, the CI tests do not confirm the fix. The python examples are not yet tested; see Issue #1294. I tested this locally though (macOS 10.12).

### CODE REVIEW

**Comment by chrisdembia** [comment]
This is a quick PR to review, if anyone is interested.

**Comment by jimmyDunne** [comment]
I'll review.

**Comment by chrisdembia** [comment]
Thanks @jimmyDunne

File: Bindings/Python/examples/build_simple_arm_model.py [hunk 1]
```diff
@@ -116,21 +116,50 @@
 reporter = osim.TableReporter()
```
Comment (jimmyDunne) [inline Bindings/Python/examples/build_simple_arm_model.py]: change to consoleReporter

**Pull request merged**